### ISSUE TITLE
mi_json doesn't set Content-Type header for "application/json"

### ISSUE THREAD

**Issue by hamid-elaosta** [OpenSIPS/opensips#1158]
```
➜ ✗ http 127.0.0.1:8888/json/get_statistics\?params=tcp-load
HTTP/1.1 200 OK
Connection: Keep-Alive
Content-Length: 22
Date: Thu, 13 Jul 2017 15:30:26 GMT

{"load:tcp-load": "0"}
```
And again with a forced json header:
```
➜ ✗ http -j 127.0.0.1:8888/json/get_statistics\?params=tcp-load
HTTP/1.1 200 OK
Connection: Keep-Alive
Content-Length: 22
Content-Type: application/json
Date: Thu, 13 Jul 2017 15:30:30 GMT

{
    "load:tcp-load": "0"
}
```
The lack of a header causes most clients to not pretty print or colourise the json response. Firefox for example with no third-party json parser demonstrates this issue.

**Comment by liviuchircu** [comment]
It's interesting how `mi_xmlrpc_ng` will reject requests without _"text/xml"_ yet it will make sure to include this same content type in its replies. OTOH, `mi_json` accepts requests regardless of the presence of content type since it only cares about the query string, and will not include a Content-Type header in its replies. Reminds me of SOAP vs. REST...

IMO, this definitely needs fixing.

**Comment by liviuchircu** [comment]
Can you try out [this](http://chunk.io/f/c00b90577ac54c6f87b0c05a55baa831) patch, and let me know if it does the trick for you?

**Comment by hamid-elaosta** [comment]
@liviuchircu Apologies. As I'm running from the apt repo and not building from source I was unable to test your patch.

I have just re-tested in 2.3.3 which still exibits this issue.

**Comment by liviuchircu** [comment]
Fixed in ff21cea1951eba922e57a1
